src/backend/metal/Cargo.toml

```diff
@@ -34,7 +34,6 @@ foreign-types = "0.3"
 objc = "0.2.5"
 block = "0.1"
 cocoa-foundation = "0.1"
-smallvec = "1"
```

Comment:
That one wasn't driven by performance. I'm just trying to narrow down the cases where smallvec was used because we didn't know what to do, versus cases where it really makes sense. The only place where it was used in Metal is `submit()`, where there is nothing serving like a natural threshold. So putting an arbitrary number in `SmallVec` wasn't nice, and it was tempting to just remove it entirely.

If we see the vecs in submit() showing up in any visible way, I'll try to find a way to solve them more aggressively.